README.md

```diff
@@ -1 +1,138 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://css3.com/">CSS</a></li>
+    <li><a href="https://javascript.com/">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+  ## The Summit Website offers the following features:
+
+* Homepage: The landing page provides an overview of the summit, highlighting key details and attracting attendees' attention.
+* About: This section offers comprehensive information about the summit, including its purpose, goals, and history.
+* Speakers: Attendees can explore the profiles of all the esteemed speakers participating in the summit.
+* Venue: Provides details about the venue, including its location, facilities, and accommodations.
+* Contact: Allows attendees to get in touch with the summit organizers for any queries or additional information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://akorede3133.github.io/oscafest-conference/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A code editor or IDE installed on your computer (e.g., Visual Studio Code, Sublime Text, Atom).
+2. Git installed on your system to clone the repository.
+3. A web browser to view and test the project.
+
+### Setup
+
+1. Open your preferred terminal or command prompt.
+2. Navigate to the directory where you want to clone the project.
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Install
+
+Install this project with:
+After cloning the repository, navigate into the project's directory. If there are any dependencies required for the project, you need to install them. Typically, this involves using a package manager like npm or yarn.
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+👤 **Akorede**
+
+- GitHub: [@githubhandle](https://github.com/Akorede3133)
+- Twitter: [@twitterhandle](https://twitter.com/SaheedAkorede7)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Attendees can register for the summit directly through the website, with an easy-to-use registration form.**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+We welcome contributions to enhance the Summit Website. If you find any issues or have suggestions for improvements, please feel free to submit a pull request or open an issue in this repository.
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] keeping your README professional means ensuring all links are working. The issue link is broken. Kindly fix that.